docroot/themes/custom/mass_theme/templates/content/node--org-page--organization-grid.html.twig

```diff
@@ -7,8 +7,7 @@
 {% set image = false %}
 {% if node.field_org_page_thumbnail is not empty %}
   {% set image = {
-    'src': file_url(node.field_org_page_thumbnail.entity.fileuri ?? ''),
-    'alt': node.field_org_page_thumbnail.alt
```

Comment:
should we also remove this field in the cms? we shouldn't allow people to input if we are not using the data right?

Reply:
I like the idea, but this field just pulls a content item that contains its image and alt text data.  In this edit page, authors don't see the alt text field.  So, we simply don't pass the alt value to the template.
<img width="567" alt="edit-page" src="https://user-images.githubusercontent.com/9633303/166709333-1b37cf4b-397f-4171-8715-a159bc476a95.png">